packages/cxl-ui/scss/global/cxl-institute-layout.scss

```diff
@@ -104,3 +104,74 @@ cxl-institute-layout {
     }
   }
 }
+cxl-institute-layout[theme~="blog-post"] {
+  > .entry {
+    .entry-header {
+      margin-bottom: var(
+        --lumo-space-xl
+      ) !important; /* stylelint-disable-line declaration-no-important */
+
+      label {
+        font-size: var(--lumo-font-size-l);
+        color: var(--lumo-primary-color);
+      }
+
+      .avatar {
+        border-radius: 50%;
```

Comment:
This is available as a global style class. Self-review for more such redundancies.